docs/faq.rst

```diff
@@ -103,6 +103,18 @@ leaving the local machine. We do not mean the transport layer encryption
 by that, but the data/metadata itself. Transport layer encryption (e.g.
 when ssh is used as a transport) applies additionally.
 
+Why are files always transferred when doing a pull backup mounting remote shares?
+---------------------------------------------------------------------------------
+
+Check that the filesystem you are mounting support reading the remote file
+inode numbers and use stable mount points (not random generated temporary
```

Comment:
I'ld say split this into 2 FAQ entries:
1) about stable inodes
2) about always using same mountpoint

Both are not necessarily specific to "mounting remote shares".